plugins/base/plugin.go

```diff
@@ -52,7 +52,9 @@ func (p *PluginBase) GRPCClient(ctx context.Context, broker *plugin.GRPCBroker,
 
 // MsgpackHandle is a shared handle for encoding/decoding of structs
 var MsgpackHandle = func() *codec.MsgpackHandle {
-	h := &codec.MsgpackHandle{RawToString: true}
+	h := &codec.MsgpackHandle{}
+	h.RawToString = true
+	h.TypeInfos = codec.NewTypeInfos([]string{"cty", "codec"})
```

Comment:
Add a comment on why

Reply:
Going to do it in #4775 because I'm too lazy to do all the rebasing just for a comment :sweat_smile: